compiler+runtime/test/jank/reader-macro/uuid/pass-random-uuid.jank

```diff
@@ -0,0 +1,3 @@
+(assert (uuid? (random-uuid)))
```

Comment:
The tests here are only for the compiler, not the runtime. We have a whole other repo of tests for the runtime (clojure.core-test). We can remove this test and change the one above it to not check `uuid?`. I think we can have more negative cases, though, such as:

* Empty string
* Valid UUID string with extra characters at the end